shared/insideResinOS/2.0.md

```diff
@@ -0,0 +1,154 @@
+# What is ResinOS
+
+* [Introduction](#introduction)
+* [Variants of ResinOS](#variants-of-resinos)
+  * [Dev vs. Prod images](#dev-vs-prod-images)
+  * [Standalone ResinOS](#standalone-resinos)
+* [ResinOS Components](#resinos-components)
+* [Stateless and Read-Only rootFS](#stateless-and-read-only-rootfs)
+* [Image Partition Layout](#image-partition-layout)
+* [OS Yocto Composition](#os-yocto-composition)
+
+## Introduction
+
+ResinOS is an operating system optimised for running Docker containers on embedded devices, with an emphasis on reliability over long periods of operation, as well as a productive developer workflow inspired by the lessons learned while building resin.io.
+
+The core insight behind resinOS is that Linux Containers offer, for the first time, a practical path to using virtualisation on embedded devices. VMs and hypervisors have lead to huge leaps in productivity and automation for cloud deployments, but their abstraction of hardware as well as their resource overhead and lack of hardware support means that they are out of the question for embedded scenarios. With OS-level virtualisation as implemented for Linux Containers, both those objections are lifted for Linux devices, of which there are many in the Internet of Things.
+
+ResinOS is an operating system built for easy portability to multiple device types (via the [Yocto framework](https://www.yoctoproject.org/)) and optimised for Linux Containers, and Docker in particular. There are many decisions, large and small, we have made to enable that vision, which are present throughout our architecture.
+
+The first version of ResinOS was developed as part of resin.io, and has run on thousands of embedded devices, deployed in many different contexts for several years. ResinOS v2 represents the combination of the learnings we extracted over those years, as well as our determination to make ResinOS a first-class open source project, able to run as an independent operating system, for any context where embedded devices and containers intersect.
+
+We look forward to working with the community to grow and mature ResinOS into an operating system with even broader device support, a broader operating envelope, and as always, taking advantage of the most modern developments in security and reliability.
+
+## Variants of ResinOS
+
+ResinOS currently comes in 3 different variants all built from the same source but with slightly differing features enabled or disabled. Each version of resinOS produces the following variants:
+
+| Version Name | Variant Type | Description |
+|:------------:|:------------:|:------------:|
+| 2.0.0.rev1   | production   | This is the production version of the resin.io managed OS. This is the OS you should use for any production fleet deployments |
+| 2.0.0.rev1-dev | development | This version the development version of the above and should be used when you are developing a new application and want to use the fast [local mode][local-mode-link] workflow |
```

Comment:
@alisondavis17 those links are referenced down the bottom of the doc, its just a markdown trick that allows you to use the same link in multiple places.